t/lib/t/Edit.pm

```diff
@@ -6,12 +6,11 @@ around run_test => sub {
     my $orig = shift;
     my $self = shift;
 
-    $self->c->sql->do(<<'EOSQL');
-INSERT INTO editor (id, name, password, ha1, email, email_confirm_date)
-VALUES
-  (1, 'editor', '{CLEARTEXT}pass', '3f3edade87115ce351d63f42d92a1834', 'foo@example.com', now()),
-  (200, 'editor200', '{CLEARTEXT}pass', '3f3edade87115ce351d63f42d92a1834', 'foo2@example.com', now());
-EOSQL
+    $self->c->sql->do(<<~'EOSQL');
+        INSERT INTO editor (id, name, password, ha1, email, email_confirm_date)
+            VALUES (1, 'editor', '{CLEARTEXT}pass', '3f3edade87115ce351d63f42d92a1834', 'foo@example.com', now()),
```

Comment:
Looking at this again, since we align SELECT, FROM, WHERE, AND, etc. all at the same level, wouldn't it make sense for INSERT and VALUES to be at the same level too?

Reply:
I dunno, as you prefer :) I guess I see FROM/WHERE as equally important than SELECT but VALUES as less important than INSERT, but I'm not too bothered by either way.

Reply:
Not sure about the relative importance, but it is true that VALUES is a standalone command (like how you can do INSERT INTO ... SELECT) whereas FROM/WHERE are tied to the SELECT.